addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -0,0 +1,267 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _, _lt
+from odoo.exceptions import UserError
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError, SERVER_URL
+
+from lxml import etree
+import base64
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountEdiFormat(models.Model):
+    _inherit = 'account.edi.format'
+
+    # -------------------------------------------------------------------------
+    # Import
+    # -------------------------------------------------------------------------
+
+    def _cron_receive_fattura_pa(self):
+        ''' Check the proxy for incoming invoices.
+        '''
+        proxy_users = self.env['account_edi_proxy_client.user'].search([('edi_format_id', '=', self.env.ref('l10n_it_edi.edi_fatturaPA').id)])
+        for proxy_user in proxy_users:
+            company = proxy_user.company_id
+            try:
+                res = proxy_user._make_request(SERVER_URL + '/iap/l10n_it_edi/1/in/RicezioneInvoice',
+                                              params={'recipient_codice_fiscale': company.l10n_it_codice_fiscale})
+            except AccountEdiProxyError as e:
+                _logger.error('Error while receiving file from SdiCoop: %s' % str(e))
```

Comment:
- `%s` *means* `str()` formatting already
- you don't need to inject parameters inside logging messages yourself, the logger API will do that for you. It's more idiomatic, and better in general as it won't do it if the log is never emitted (depending on log level)
- you may want to consider using `_logger.exception()` if the traceback has a chance to be relevant (here it may not be the case)

The same applies everywhere in this PR, I've seen several occurrences in passing ;-)

```suggestion
                _logger.error('Error while receiving file from SdiCoop: %s', e)
```